src/anonymousUserTracking/criteriaCompletionChecker.ts

```diff
@@ -247,108 +220,32 @@ class CriteriaCompletionChecker {
 
   private evaluateSearchQueries(
     node: SearchQuery,
-    localEventData: any[],
-    criteriaId: string
+    localEventData: any[]
   ): boolean {
     // this function will compare the actualy searhqueues under search combo
     for (let i = 0; i < localEventData.length; i++) {
       const eventData = localEventData[i];
       const trackingType = eventData[SHARED_PREFS_EVENT_TYPE];
       const dataType = node.dataType;
-      if (!Object.prototype.hasOwnProperty.call(eventData, 'criteriaId')) {
```

Comment:
Can you elaborate a bit on why this is removing so much code/functionality? The ticket linked is only supposed to be for adding the `NOT` combinator logic.

Reply:
Did it get moved to `evaluateEvent`?